test/helpers/kubectl.go

```diff
@@ -210,6 +211,22 @@ func (kub *Kubectl) CiliumEndpointsList(pod string) *CmdRes {
 	return kub.CiliumExec(pod, "cilium endpoint list -o json")
 }
 
+//CiliumEndpointsIDs returns a mapping with of a pod name to it is
```

Comment:
Incorrect grammar: "returns a mapping of a pod name to its corresponding endpoint ID"